Resources/Prototypes/_White/Catalog/uplink_catalog.yml

```
@@ -82,6 +82,17 @@
   - UplinkImplants
 
+- type: listing
+  id: UplinkMindSlaveImplanter
+  name: uplink-mind-slave
+  description: uplink-mind-slave-desc
+  icon: { sprite: /Textures/_White/Overlays/mindslave.rsi, state: master }
+  productEntity: MindSlaveImplanter
+  cost:
+    Telecrystal: 6
+  categories:
+  - UplinkImplants
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)